deploy/aws-lambda/webhook.tf

```diff
@@ -0,0 +1,116 @@
+data "github_release" "webhook" {
+  repository  = "secrethub-kubernetes-mutating-webhook"
+  owner       = "secrethub"
+  retrieve_by = "latest"
```

Comment:
We should source this from a `variable` and pin it to `v0.2.0`.

Reply:
Adjusted to using tag version

Reply:
Where? Did you push the change?

Reply:
I forgot. My bad. now it's pushed